packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.ts

```diff
@@ -45,25 +45,58 @@ import {
 import { ForestType, type ISharedTreeEditor } from "../../../shared-tree/index.js";
 import {
 	MockTreeCheckout,
+	checkoutWithContent,
 	flexTreeViewWithContent,
 	numberSequenceRootSchema,
+	testIdCompressor,
 } from "../../utils.js";
+import { SchemaFactory, TreeConfiguration, toFlexConfig } from "../../../simple-tree/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { toFlexSchema } from "../../../simple-tree/toFlexSchema.js";
+// eslint-disable-next-line import/no-internal-modules
+import type { StableId } from "../../../../../../runtime/id-compressor/dist/index.js";
```

Comment:
@taylorsw04, tree needs to clean up their layers. We should not have any import/no-internal-modules disables. tree has 204 hits.
This one lead to performance tests being broken for 6 days. (Also why don't pipeline breaks get noticed.)

Filed AB#8286 - needs it import through `@fluidframework/id-compressor`.

Reply:
I noticed the same thing and was about to point it out, glad to see someone else noticed. We have AB#6636 to set up monitoring for this pipeline. For a while it was relatively unstable so we didn't want the noise, but I think it's worth adding now.